vulnerabilities/importer_yielder.py

```diff
@@ -97,13 +97,13 @@
         "data_source": "RetireDotnetDataSource",
         "data_source_cfg": {"repository_url": "https://github.com/RetireNet/Packages.git"},
     },
-    {
-        "name": "suse_backports",
-        "license": "",
-        "last_run": None,
-        "data_source": "SUSEBackportsDataSource",
-        "data_source_cfg": {"url": "http://ftp.suse.com/pub/projects/security/yaml/", "etags": {}},
-    },
+    # {
```

Comment:
Why is this disabled?

Reply:
We probably want to get rid of that importer, since it only contains `(vulnerability, fixed/backported package)`